### ISSUE TITLE
Can we conduct academic research on objects where "Is Public Domain" column is False?

### ISSUE THREAD

**Issue by fangyizhu** [metmuseum/openaccess#50]

**Comment by spencerk** [comment]
Yes, you can.  The dataset is free to use, only the non-public domain images are restricted.  https://github.com/metmuseum/openaccess#the-metropolitan-museum-of-art-open-access-csv

Thanks for your interest!
